exercises/list-ops/canonical-data.json

```diff
@@ -224,11 +421,72 @@
           "description": "direction dependent function applied to non-empty list",
           "property": "foldr",
           "input": {
-            "list": [2, 5],
+            "list": [
+              2,
+              5
+            ],
             "initial": 5,
             "function": "(x, y) -> x / y"
           },
           "expected": 2
+        },
+        {
+          "uuid": "17214edb-20ba-42fc-bda8-000a5ab525b0",
+          "reimplements": "aeb576b9-118e-4a57-a451-db49fac20fdc",
+          "comments": [
+            "Reimplemented to remove ambiguity about the parameters of the    ",
+            "function input."
+          ],
+          "description": "empty list",
+          "property": "foldr",
+          "input": {
+            "list": [],
+            "initial": 2,
+            "function": "(acc, el) -> el * acc"
+          },
+          "expected": 2
+        },
+        {
+          "uuid": "e1c64db7-9253-4a3d-a7c4-5273b9e2a1bd",
+          "reimplements": "c4b64e58-313e-4c47-9c68-7764964efb8e",
+          "comments": [
+            "Reimplemented to remove ambiguity about the parameters of the    ",
+            "function input."
+          ],
+          "description": "direction independent function applied to non-empty list",
+          "property": "foldr",
+          "input": {
+            "list": [
+              1,
+              2,
+              3,
+              4
+            ],
+            "initial": 5,
+            "function": "(acc, el) -> el + acc"
+          },
+          "expected": 15
+        },
+        {
+          "uuid": "8066003b-f2ff-437e-9103-66e6df474844",
+          "reimplements": "be396a53-c074-4db3-8dd6-f7ed003cce7c",
+          "comments": [
+            "Reimplemented to remove ambiguity about the parameters of the    ",
+            "function input."
+          ],
+          "description": "direction dependent function applied to non-empty list",
+          "property": "foldr",
+          "input": {
+            "list": [
+              1,
+              2,
+              3,
+              4
+            ],
+            "initial": 24,
+            "function": "(acc, el) -> el / acc"
+          },
+          "expected": 64
```

Comment:
By my understanding of foldr, this should be 9 instead. Can you confirm?

* acc = 24, el = 4. new acc = 1/6
* acc = 1/6, el = 3. new acc = 18
* acc = 18, el = 2. new acc = 1/9
* acc = 1/9, el = 1. new acc = 9

Reply:
It's 9 in my test suite 🙈 . You're correct.